### DISCUSSION TITLE
Functional programming in scan

### DISCUSSION THREAD

**Discussion by jazznbass** [jazznbass/scan#13] [Ideas]
Add functions:

- select_cases() [a basic structure is already in place in scan, but should be extended to select cases not just by name or number but by criteria from a level-2 dataset]
- select_measurements() [e.g.:  scdf %>% select_measurement(teacher_present == 1 & depression < 5)  ]
- select_variables() [e.g.: scdf %>% select_variables(values, mt, phase, depression)
- select_phases() [e.g.: scdf %>% select_phases(A2, B2) ; scdf %>% select_phases(A = c("A1", "A2"), B = c("B1", "B2")]

**Comment by researchtool** [comment]
we talked about ways to inform users about conflicts / masked functions. Have you seen this one?
![fnctn_msk](https://user-images.githubusercontent.com/5195357/108480994-e99de780-7297-11eb-8e61-e8553c07ad3c.PNG)
something like this with an additional "plain language what do i need to know" might me cool.

**Comment by jazznbass** [comment]
moved this Idea to a new issue
